ga1_draft.py

```diff
@@ -0,0 +1,52 @@
+from __future__ import division
+import numpy as np  
+import matplotlib.pyplot as plt
+from pymatgen import MPRester 
+
+# import/truncate raw data (volume, density, energies, etc)
+	m = MPRester("ID")
```

Comment:
true, could be more elegant, though (c.f. [this](https://gist.github.com/tschaume/fa8d0fc7ed29fe972d2c#file-pymatgen_rest_test-py-L5-L7)).